extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -578,6 +578,8 @@ namespace Microsoft.Windows.DevHome.SDK
         DeleteSnapshot = 0x00000400,
         ApplyConfiguration = 0x00000800,
         ModifyProperties = 0x00001000,
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 4)] PinToStartMenu = 0x00002000,
+        [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 4)] PinToTaskbar = 0x00004000,
```

Comment:
I know this is probably just VS idl formatting but would it be possible to put these on a new line, the new enums I mean

Reply:
Yep, I've changed it. That does look better